helm/cluster-azure/values.schema.json

```diff
@@ -160,6 +205,17 @@
             "type": "object"
         },
         "machineDeployments": {
+            "default": [
+                {
+                    "customNodeLabels": [],
+                    "customNodeTaints": [],
```

Comment:
At a closer look, I think these defaults are not needed, as they are empty anyway.